vhdl_lang/Cargo.toml

```diff
@@ -25,6 +25,7 @@ parking_lot = "0"
 dunce = "1"
 pinned_vec = "0"
 itertools = "0"
+serde = { version = "1.0.156", features = ["derive"] }
```

Comment:
It feels a bit overkill to add a huge crate like serde as a dependency to `vhdl_lang` just to be able to serialize EntityId. Maybe Add a `toRaw/fromRaw`  method and let serde stay a dependency only in `vhdl_ls` where it is actually needed.